src/5e-SRD-Magic-Items.json

```diff
@@ -21,12 +25,86 @@
       "name": "Ammunition",
       "url": "/api/equipment-categories/ammunition"
     },
+    "rarity": {
+      "name": "Varies"
+    },
+    "variants": [
+      {
+        "index": "ammunition-1",
+        "name": "Ammunition, +1",
+        "url": "/api/magic-items/ammunition-1"
+      },
+      {
+        "index": "ammunition-2",
+        "name": "Ammunition, +2",
+        "url": "/api/magic-items/ammunition-2"
+      },
+      {
+        "index": "ammunition-3",
+        "name": "Ammunition, +3",
+        "url": "/api/magic-items/ammunition-3"
+      }
+    ],
     "desc": [
       "Weapon (any ammunition), uncommon (+1), rare (+2), or very rare (+3)",
       "You have a bonus to attack and damage rolls made with this piece of magic ammunition. The bonus is determined by the rarity of the ammunition. Once it hits a target, the ammunition is no longer magical."
     ],
     "url": "/api/magic-items/ammunition"
   },
+  {
+    "index": "ammunition-1",
+    "name": "Ammunition, +1",
+    "equipment_category": {
```

Comment:
We'll need to updated [Equipment Categories](https://github.com/5e-bits/5e-database/blob/main/src/5e-SRD-Equipment-Categories.json) as well.

Reply:
updated